components/Common/CodeBox/CodeTabs/index.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { FC } from 'react';
+
+import { CodeBox } from '@/components/Common/CodeBox';
+import { CodeTabs } from '@/components/Common/CodeBox/CodeTabs';
+
+type Story = StoryObj<typeof CodeTabs>;
+type Meta = MetaObj<typeof CodeTabs>;
+
+const TabsContent: FC = () => (
+  <>
+    <TabsPrimitive.Content key="mjs" value="mjs">
+      <CodeBox language="JavaScript (MJS)">
+        <code>
+          {`import * as http from 'http';
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World');
+});
+
+server.listen(port, hostname, () => {
+  console.log(\`Server running at http://\${hostname}:\${port}/\`);
+});`}
+        </code>
+      </CodeBox>
+    </TabsPrimitive.Content>
+    <TabsPrimitive.Content key="cjs" value="cjs">
+      <CodeBox language="JavaScript (CJS)">
+        <code>
+          {`const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World');
+});
+
+server.listen(port, hostname, () => {
+  console.log(\`Server running at http://\${hostname}:\${port}/\`);
+});`}
+        </code>
+      </CodeBox>
+    </TabsPrimitive.Content>
+  </>
+);
+
+export const Default: Story = {};
+
+export const WithMoreOptions: Story = {
+  args: {
+    linkUrl: 'https://github.com/nodejs/nodejs.org',
+    linkText: 'More options',
+  },
+};
+
+export default {
+  component: CodeTabs,
+  args: {
+    children: <TabsContent />,
+    defaultValue: 'mjs',
+    tabs: [
+      {
```

Comment:
```suggestion
      { key: 'mjs', label: 'MJS' }
```

We prefer inline objects when possible, when Prettier alows it, just to use less lines